recipes/px4tools/meta.yaml

```diff
@@ -0,0 +1,95 @@
+{% set name = "px4tools" %}
+{% set version = "0.3.7" %}
+{% set sha256 = "165351d235869e29449003c1bd4b36c64561f85e1932f0a12e739a25dbfc8227" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
```

Comment:
Please change to `{{ name }}-{{ version }}.tar.gz`.

This improves source caching for people building locally.